packages/generator-one-app-module/generators/app/templates/base-module/src/components/ModuleContainer.jsx

```diff
@@ -15,9 +15,7 @@ const <%=moduleNamePascal%> = () => (
 
 // Read about appConfig: https://github.com/americanexpress/one-app#appconfig
 if (!global.BROWSER) {
-  <%=moduleNamePascal%>.appConfig = {
-    csp,
-  };
+  <%=moduleNamePascal%>.appConfig = appConfig;
```

Comment:
```suggestion
  <%=moduleNamePascal%>.appConfig = require('../appConfig');
```

Reply:
eslint doesn't like `require` not being at the top-level, so I'd have to throw in the disable rule above the line. Is this preferable to importing the appConfig at top-level?

Reply:
fixed in 6ca9aa7